src/main/java/com/keeper/homepage/domain/member/application/MemberService.java

```diff
@@ -60,4 +64,11 @@ public void unfollow(Member member, long otherId) {
     Member other = memberFindService.findById(otherId);
     member.unfollow(other);
   }
+
+  public MemberProfileResponse getMemberProfile(long memberId) {
+    Member findMember = memberFindService.findById(memberId);
+    List<Friend> findAllFollowers = friendRepository.findAllByFollowee(findMember);
+
+    return MemberProfileResponse.of(findMember, findAllFollowers);
```

Comment:
```suggestion
    Set<Friend> friends = findMember.getFollower();
    List<Member> follower = friends.stream().map(Friend::getFollower).toList();
    List<Member> followee = friends.stream().map(Friend::getFollowee).toList();

    return MemberProfileResponse.of(findMember, follower, followee);
```

이렇게 팔로워와 팔로잉 멤버 리스트를 뽑아서 DTO로 전달하는 건 어떨까요..?

예전에 `Friends` 엔티티 만들때 저도 많이 헷갈려서 고생하다가 아래처럼 `follower`와 매핑된 `friends` set 필드 하나만 두었었는데 ([레파지토리 테스트 코드](https://github.com/KEEPER31337/Homepage-Back-R2/blob/develop/src/test/java/com/keeper/homepage/domain/member/dao/friend/FriendRepositoryTest.java))
```java
@OneToMany(mappedBy = "follower", cascade = ALL, orphanRemoval = true)
private final Set<Friend> friends = new HashSet<>();
```

고스란히 까먹고 회원 팔로우, 언팔로우 기능 만들 때 이름도 다시 헷갈리게 바꾸고 (`follower`로) ... `follower`도 끼워넣었네요...